src/test_pixel_gap_cuts.cpp

```diff
@@ -43,11 +44,12 @@ int main(int argc, char** argv) {
     return 1;
   }
   auto compactFile = DETECTOR_PATH + "/" + DETECTOR_CONFIG + ".xml";
-  auto det = &dd4hep::Detector::getInstance();
-  det->fromXML(compactFile);
+  dd4hep::Detector & det = dd4hep::Detector::getInstance();
+  det.fromXML(compactFile);
+  dd4hep::rec::CellIDPositionConverter cellid_converter(det);
 
   // ReadoutGeo
-  richgeo::ReadoutGeo geo("DRICH", det, logger);
+  richgeo::ReadoutGeo geo("DRICH", gsl::not_null{ &det}, gsl::not_null{ &cellid_converter}, logger);
```

Comment:
gsl::not_null unneeded for address of object: always not_null.
```suggestion
  richgeo::ReadoutGeo geo("DRICH", &det, &cellid_converter, logger);
```